azure_monitor/src/azure_monitor/sdk/auto_collection/live_metrics/__init__.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+import time
+import typing
+import uuid
+
+from azure_monitor.protocol import BaseObject
+from azure_monitor.utils import azure_monitor_context
+
+LIVE_METRICS_SUBSCRIBED_HEADER = "x-ms-qps-subscribed"
+STREAM_ID = str(uuid.uuid4())
+
+
+def create_metric_envelope(instrumentation_key: str):
+    envelope = LiveMetricEnvelope(
+        documents=None,
+        instance=azure_monitor_context.get("ai.cloud.roleInstance"),
+        instrumentation_key=instrumentation_key,
+        invariant_version=1,  # 1 -> v1 QPS protocol,
+        machine_name=azure_monitor_context.get("ai.device.id"),
+        metrics=None,
+        stream_id=STREAM_ID,
+        timestamp="/Date({0})/".format(str(int(time.time()) * 1000)),
+        version=azure_monitor_context.get("ai.internal.sdkVersion"),
+    )
+    return envelope
+
+
+class LiveMetricDocumentProperty(BaseObject):
```

Comment:
Can we put these schema classes into `protocol.py`?

Reply:
My reasoning was that these classes will never be relevant to users because they are internal, not sure if putting them in common place could cause confusion with users

Reply:
What do you mean by `internal`? All classes in `protocol.py` are `internal` in the fact that they are all Microsoft specific.

Reply:
Yes, but that could change in the case of we adding APIs for trackEvents in SDK, then users could be using the protocols to send the telemetry, well that was my reasoning when added the code in different file, will put everything in protocols.py for now and we can move later if it makes sense